### DISCUSSION TITLE
Welcome to Marp Discussions!

### DISCUSSION THREAD

**Discussion by yhatt** [marp-team/marp#40] [General]
## 👋 Welcome to Marp Discussions!

Hi, we're using GitHub Discussions as a place to connect with others of Marp community. 

* Ask any questions about using Marp.
* Share your Marp deck, theme, and any kind of ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yhatt** [comment]
It's first introduce! I'm Yuki Hattori, the author and maintainer of Marp ecosystem, frontend developer, and open-source lover. I'm Japanese and unfortunatelly English is not good at all. Thanks to translator ;)

I had created an original Marp desktop app (https://yhatt.github.io/marp/) about 4 years ago and luckily has supported by many users. Now I've developed new Marp ecosystem and keeping to maintain to save the time for people who creates the deck.

I would respond questions from community too. I hope this forum will be the best place for community to collect tips about Marp.

**Comment by StEvUgnIn** [comment]
Congratulations! Your work is amazing 🔥✨. I am already in ❤ with marp

**Comment by frigaut** [comment]
Hello Yuki,

Francois here, I have been looking/investigating various markdown slide deck generation tools in the past couple of weeks, motivated by the fact I wanted to refresh the lecture I am giving in optical physics (quite a bit of equations, which makes using the standard tools, powerpoint, keynote and the like, quite clunky). 

I have been looking at a number of reveal.js interface tools, like the one provided by Markdown-preview-enhanced (a very good tool), and reveal-md. These are generally fairly good, and they are doing the trick for me, but it's quite a bit of a stack if you think about it (MPE is huge, reveal.js is huge too). And then there are version compatibility, etc. None of them was doing exactly all that I wanted, i.e. good support for Markdown, Math, images, export to PDF (and html of course), and convenient presentation features.

Then I stumbled on Marp. I like it pretty good thus far.  I might well make it my to-go presentation package. IMO, it's just lacking a couple of useful features like overview mode (or grid mode) when doing the presentation, and something equivalent to chalkboard or highlight that you find in reveal.js and that it quite useful when teaching.

Anyway, congratulations, your work is amazing.

**Reply by yhatt** [reply]
Hi Francois, thank you for your voice!

Marp keeps looking for a balanced way for creating the Markdown presentation, with easy and useful interfaces for most users, and the primitive approach for the ecosystem's long life and extensibility.

We are aiming to minimal and practical, so fancy features like you mentioned are not yet ready. I would be grateful if you are looking your eyes on the updation of our tools and community!

**Reply by frigaut** [reply]
Thanks Yuki.
Indeed I agree Marp looks solid and certainly one important feature is stability/long life, and I agree too bloated is not good.
Francois

**Comment by menzzana** [comment]
Hello

My name is Henric Zazzi, and I work within academia around HPC computer. In my line of work I create presentation for courses etc
as to help user understand how use HPC infrastructure for analysis.
I always liked the idea of creating MD presentation, as they are easier version controlled for future versions of the presentations I am creating.
I have used gitpitch which I though ticked a lot of boxes when creating these MD presentations but am now looking at Marp, which I think is really nice, open source.
What I am missing in marp is the ability to create several objects/boxes/row in one slide, aka split the screen in various ways. That
was something that is possible with gitpitch.
One way I tried to achieve the same functionality is to add a HTML table to the slide, and then write, or add pictures, to the different cells in the table. A table also has the advantage to create as many cells you want to you can split your slide quite a lot.
A problem though is even if I say that the border=0 I still see a miniscule border on the slide.
Overall, A similar split functionality created in MD would be very nice, but I have seen that you do not want to stray so far from MD standards.
